Bottom: cf7afed33de3a862454c018e593ef83e27d4ae10
Top:    7e212b135f866cfb88cc1b752e6405a5f5235d57
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-27 22:03:43 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
